.changeset/quiet-paws-rush.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin": patch
+---
+
+Prevent a bug that would cause the page tree to have no height when navigating into a page and then navigating back again.
```

Comment:
```suggestion
Prevent the `MainContent` component from having an invalid height. 

This bug would cause the page tree to have no height when navigating into a page and then navigating back again.
```